src/components/Lesson/Lesson.js

```diff
@@ -81,6 +98,8 @@ const Lesson = React.createClass({
     const instructionButton = isReadme ? <LessonButton {...{path}}/> :
       isStudentMode ? null : <ReadmeButton {...{path}}/>;
     const pdfButton = <PdfButton lessonfile={params.file}/>;
+    const progress = (checkedCheckboxes > 0 && isStudentMode && !isReadme) ?
```

Comment:
Er vi sikre på at vi ikke vil ha progressbar i lærermodus også?

Reply:
Hm, nå ble jeg usikker. For i første omgang tenkte jeg at det ikke var noen vits, men jo mer jeg tenker over det, kan de like godt få ha det de også. Men lærerveiledninger trenger ikke. Og så klart hvis det ikke er huket noen ting av.